packages/neuron-wallet/src/services/tx/indexer-transaction.ts

```diff
@@ -61,10 +61,11 @@ export default class IndexerTransaction {
       .where({
         outPointTxHash: txHash,
         outPointIndex: index,
+        status: Not(OutputStatus.Dead)
```

Comment:
This might be lower since we don't have index on `[outPointTxHash, outPointIndex, status]` or `[status]`.

`(outPointTxHash, outPointIndex)` is primary key, I believe the previous query before this change is faster and cost less (without having to create a new index).

Reply:
I think we should test it with / without index, to consider rollback or add an index.
If add `status` to query conditions, it will not load data to memory if `status != dead`, it will save some costs.

Reply:
I have tested in `test` env ( in a test case).

I created 10k `outputs` in sqlite memory db. (half `live` and half `dead`)

`with status Not(dead)` | `with status Not(dead) and without index` | `only primary key` | `add union index and with status` 
--- | --- | --- | ---
query 1k times (shuffled and first 1k txs) | `360ms` | `318ms` | `291ms`
query 10k times (shuffled and load each tx) | `2667ms` | `2278ms` | `2249ms`
query 100k times (shuffled and load loop 10 times) | `21083ms` | `22047ms` | `20227ms`

And tested created 10k Outputs

`without index` | `with index`
--- | ---
`3192ms` | `3189ms`

Reply:
Only use primary key is faster than with status but no index, add an index to `outPointTxHash, outPointIndex, status` seems make limited promotion

I'll rollback to query only using `outPointTxHash, outPointIndex`